docs/source/install.rst

```diff
@@ -129,7 +131,7 @@ The installation instructions vary depending on your package manager:
 
       .. code-block:: bash
 
-	  guix install python-cocotb
+	    guix install python-cocotb
```

Comment:
I guess the tab is the actual problem here that you tried to solve.

Reply:
Yeah it's a little busted.